### PR TITLE
A bit more protocol documentation

### PR DESCRIPTION
**Pull request by da-kami**
When finalising my review on https://github.com/comit-network/spikes/blob/communication-protocol/0023-communication-protocol.adoc I felt I could bring some of the knowledge there into the docu quickly. 

Note: I am not totally sure about the abstraction level of this part of the documentation yet, but I would add information where possible and then collect feedback from people using the documentation. 

I'd encourage other team members to do the same if the notice such kind of things :)

### CODE REVIEW

File: docs/comit-protocol/comit-protocol-stack.md [hunk 1]
```diff
@@ -23,24 +23,61 @@ This slide shows the motivation of the COMIT protocol that is to create a truly
...
+* HAN for ether on Ethereum
+
+#### HErc20
 
 **H**TLCs for the **Erc20** asset
 
-### HALight
+The HErc20 protocol supports the following asset:
+
+* HErc20 tokens on Ethereum
```
Comment (thomaseizinger) [inline docs/comit-protocol/comit-protocol-stack.md]:
```suggestion
* Erc20 tokens on Ethereum
```

File: docs/comit-protocol/comit-protocol-stack.md [hunk 2]
```diff
@@ -23,24 +23,61 @@ This slide shows the motivation of the COMIT protocol that is to create a truly
...
+Those (libp2p-based) protocols facilitate the exchange of information for the purpose of then following a cryptographic protocol.
+
+### COMIT Cryptographic Protocols
+
+The COMIT protocol distinguishes between ledgers and assets.
+When referring to a ledger we use a an uppercase first letter (e.g. Bitcoin, Ethereum), when referring to an asset we use a lowercase first letter (e.g. bitcoin, ether).
+
+COMIT currently defines the following atomic swap protocols:
+
+* HAN - **H**TLCs for **A**ssets that are **N**ative to the ledger.
```
Comment (bonomat) [inline docs/comit-protocol/comit-protocol-stack.md]: There are no . at the end of the other two points
```suggestion
* HAN - **H**TLCs for **A**ssets that are **N**ative to the ledger
```

File: docs/comit-protocol/comit-protocol-stack.md [hunk 3]
```diff
@@ -23,24 +23,61 @@ This slide shows the motivation of the COMIT protocol that is to create a truly
...
+* HAN for ether on Ethereum
+
+#### HErc20
 
 **H**TLCs for the **Erc20** asset
 
-### HALight
+The HErc20 protocol supports the following asset:
+
+* Erc20 tokens on Ethereum
```
Comment (bonomat) [inline docs/comit-protocol/comit-protocol-stack.md]:
```suggestion
* [Erc20](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md) tokens on Ethereum
```

File: docs/comit-protocol/comit-protocol-stack.md [hunk 4]
```diff
@@ -23,24 +23,61 @@ This slide shows the motivation of the COMIT protocol that is to create a truly
...
 
 **H**TLCs for **A**ssets on the **Light**ning ledger
 
-## COMIT Messaging Protocols
+The HALight protocol is currently still in development, the first supported asset will be bitcoin on the Lightning network.
+
+## COMIT Communication Protocols
 
-The messaging protocol of COMIT is used to exchange information between parties prior to the execution of a [swap](../core-concepts/atomic-swap-htlc.md).
+The communication protocols of COMIT is are used to exchange information between parties prior to the execution of a [swap](../core-concepts/atomic-swap-htlc.md).
```
Comment (bonomat) [inline docs/comit-protocol/comit-protocol-stack.md]:
```suggestion
The communication protocols of COMIT are used to exchange information between parties prior to the execution of a [swap](../core-concepts/atomic-swap-htlc.md).
```

**Pull request merged**